src/app/fdctl/keygen.c

```diff
@@ -67,4 +67,5 @@ keygen_cmd_fn( args_t *         args,
   (void)args;
 
   generate_keypair( config->consensus.identity_path, config );
+  generate_keypair( config->consensus.vote_account_path, config );
```

Comment:
I think this command should only generate one key at a time. Can you just add a new mandatory argument,

`keygen <purpose>`

purpose is one of `identity`, `vote_account`, etc.

Reply:
done